src/Akeneo/Pim/Enrichment/Component/Product/Factory/NonExistentValuesFilter/NonExistentMultiSelectValuesFilter.php

```diff
@@ -91,6 +92,17 @@ private function getOptionCodes(array $selectValues): array
 
         $uniqueOptionCodes = [];
         foreach ($optionCodes as $attributeCode => $optionCodeForThisAttribute) {
+            foreach ($optionCodeForThisAttribute as $value) {
+                if (!is_scalar($value)) {
```

Comment:
```suggestion
                if (!\is_scalar($value)) {
```

Everywhere? As we know it impact the perf more than exepcted.